CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 4.5.1 (2021-03-26)
+
+- Fixed the action button container component to be center aligned by default. [#201](https://github.com/blackbaud/skyux-layout/pull/201)
+- Fixed the action button container component to have `20px` margins in modern theme. [#200](https://github.com/blackbaud/skyux-layout/pull/200)
+- Fixed the action button component to no longer throw null injector errors for `SkyThemeService`. [#200](https://github.com/blackbaud/skyux-layout/pull/200)
```

Comment:
```suggestion
- Fixed the action button component to prevent null injector errors for `SkyThemeService` and `SkyCoreAdapterService `. [#200](https://github.com/blackbaud/skyux-layout/pull/200)
```
Not sure if `SkyCoreAdapterService ` is necessary, but we mention it in the PR, so it seemed like we might should call out both.